src/applications/apps.js

```diff
@@ -15,6 +15,11 @@ export function getAppNames() {
   return apps.map(toName);
 }
 
+// used in devtools, not (currently) exposed as a single-spa API
+export function getRawAppData() {
+  return [...apps];
```

Comment:
I still have some worry about exposing the full internal representation to the dev tools, because we can't call the methods on them anyways, I'm still interested in these values being a copy of the internal representation long term.